src/components/popover-container/popover-container.component.js

```diff
@@ -92,6 +99,8 @@ const PopoverContainer = ({
             shouldCoverButton={shouldCoverButton}
             aria-labelledby={popoverContainerId}
             aria-describedby={ariaDescribedBy}
+            p="16px 24px"
```

Comment:
I think that `PopoverContainerContentStyle` default props would be a better place for this